### PR TITLE
task solution #7

### PR DESCRIPTION
**Pull request by NasyTheSlayer**
[DEMO LINK](https://NasyTheSlayer.github.io/layout_product-cards/)
[TEST REPORT LINK](https://NasyTheSlayer.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,53 @@
...
+        class="card__img"
+      >
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__product-code">
+        Product code: 195434
+      </p>
+      <div class="card__stars-box">
+        <div class="stars stars--4">
```
Comment (vlad-demchuk) [inline src/index.html:25]:
```suggestion
    <div class="card" data-qa="card">
      <img
        src="./images/imac.jpeg"
        alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
        class="card__img"
      >
      
      <h2 class="card__title">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h2>
      
      <p class="card__product-code">
        Product code: 195434
      </p>
      
      <div class="card__stars-box">
        <div class="stars stars--4">
```
checklist
[CODE STYLE] - Add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements

Comment (vlad-demchuk) [reply src/index.html:25]: fix all such cases

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
+body,
+div,
+h2,
+p {
+  margin: 0;
```
Comment (vlad-demchuk) [inline src/style.css:5]: better to move to `reset.css`

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+}
+
+.card {
```
Comment (vlad-demchuk) [inline src/style.css:15]: checklist
[FILE STRUCTURE] - You should create separate file for each BEM block

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+  box-sizing: border-box;
```
Comment (vlad-demchuk) [inline src/style.css:16]: move `box-sizing` to `html` selector

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
```
Comment (vlad-demchuk) [inline src/style.css:21]: css properties order
http://codeguide.academy/html-css.html#css-order

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+  margin: 0 0 4px;
```
Comment (vlad-demchuk) [inline src/style.css:36]: try to use only `margin-bottom` property

if some elements have default margins you can reset it for tag

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars {
```
Comment (vlad-demchuk) [inline src/style.css:57]: separate file per bem block

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+  display: flex;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
```
Comment (vlad-demchuk) [inline src/style.css:61]: it should work like in the previous task
task description
image

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]

File: src/blocks/stars.css [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(/images/star-active.svg);
+}
+
+.card__stars-box {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
```
Comment (DarkMistyRoom) [inline src/blocks/stars.css:27]: move it to the card.css

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
Good job, let's improve it:
better place all styles in one folder
image

File: src/styles/blocks/card.css [hunk 10]
```diff
@@ -0,0 +1,92 @@
+* {
```
Comment (YegorKorenkov) [inline src/styles/blocks/card.css:1]: Please, also move it to reset.css

**Review by YegorKorenkov — APPROVED** [review]
Everething okey, but fix one mistake

**Pull request closed without merging**